lib/updateFacebookCache.js

```diff
@@ -1,15 +1,30 @@
 var axios = require('axios');
 var config = require('../config/config.json');
 
-// update facebook's cache with most recent metadata
-function updateFacebookCache ( url ) {
-	axios.post('https://graph.facebook.com', { id: url, scrape: true, access_token: config.fbAccessToken })
-	  .then(function (response) {
-	    console.log(response);
-	  })
-	  .catch(function (error) {
-	    console.log(error);
-	  });
+function updateFacebookCache (schema) {
+
+	schema.Metadata.findAll({
+	    attributes: [
+	        [Sequelize.fn('DISTINCT', Sequelize.col('url')) ,'url'],
+	    ]
+	}).then(staleUrls => {
+		if (staleUrls) {
+			staleUrls.forEach((entry) => {
+				axios.post('https://graph.facebook.com', { 
+					id: 'https://petitions.moveon.org/sign/rep-faso-owned-by-nra?source=homepage', scrape: true, access_token: config.fbAccessToken 
```

Comment:
`id:` probably needs to be the constructed *sharebandit* url `baseUrl + '/r/0/' +entry.dataValues.url` or something?